pkg/collectors/pool.go

```diff
@@ -291,60 +431,128 @@ func isParentPool(pool Pool) bool {
 	if isParentPool(pool) {
+		var reclaimableCalculatedCapacity float64
 		physicalFree, err := strconv.ParseFloat(pool[PhysicalFreeKey].(string), 64)
 		if err != nil {
 			log.Errorf("get physical free failed: %s", err)
 			return
 		}
-
-		reclaimable, err := strconv.ParseFloat(pool[ReclaimableKey].(string), 64)
+		physical, err := strconv.ParseFloat(pool[PhysicalCapacityKey].(string), 64)
 		if err != nil {
-			log.Errorf("get reclaimable failed: %s", err)
+			log.Errorf("get physical capacity failed: %s", err)
 			return
 		}
-
-		physical, err := strconv.ParseFloat(pool[PhysicalCapacityKey].(string), 64)
+		poolOrigReclaimable, err := strconv.ParseFloat(pool[ReclaimableKey].(string), 64)
 		if err != nil {
-			log.Errorf("get physical capacity failed: %s", err)
+			log.Errorf("get reclaimable failed: %s", err)
 			return
-
 		}
-		usable := physicalFree + reclaimable
-		used := physical - usable
+		if poolOrigReclaimable != 0 {
+			reclaimableCalculatedCapacity, err = f.GetPoolReclaimablePhysicalCapacity(pool, fsRestClient, mDisksList)
+			if err != nil {
+				log.Errorf("get reduced reclaimable capacity failed: %s", err)
+				return
+			}
+		} else {
+			reclaimableCalculatedCapacity = 0
+		}
+
+		used := physical - physicalFree - reclaimableCalculatedCapacity
+		usable := physicalFree + poolOrigReclaimable
+
 		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolCapacityUsable], usable, &poolInfo)
 		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolCapacityUsed], used, &poolInfo)
+		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolPhysicalCapacity], physical, &poolInfo)
 	} else {
 		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolCapacityUsable], float64(-1), &poolInfo)
 		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolCapacityUsed], float64(-1), &poolInfo)
+		newPoolCapacityMetrics(ch, f.poolDescriptors[PoolPhysicalCapacity], float64(-1), &poolInfo)
+	}
+}
+
+func (f *PerfCollector) GetPoolReclaimablePhysicalCapacity(pool Pool, fsRestClient *rest.FSRestClient, mDisksList rest.MDisksList) (float64, error) {
+	var reclaimable float64
+	dataReduction := pool[DataReductionKey].(string)
+
+	compressionEnabled, err := isCompressionEnabled(pool[MdiskNameKey].(string), mDisksList, fsRestClient)
+	if err != nil {
+		log.Errorf("Failed to determine internal or external pool: %s", err)
+		return reclaimable, err
+	}
+
+	internalPool, err := isPoolInternalMDisks(pool[MdiskNameKey].(string), mDisksList)
+	if err != nil {
+		log.Errorf("Failed to determine internal or external pool: %s", err)
+		return reclaimable, err
+	}
+	arrayMode, err := isPoolArrayMode(pool[MdiskNameKey].(string), mDisksList)
+	if err != nil {
+		log.Errorf("Failed to determine disk mode: %s", err)
+		return reclaimable, err
+	}
+
+	if compressionEnabled && dataReduction == "yes" && internalPool && arrayMode {
```

Comment:
consider moving `dataReduction == "yes"` to line 520 (so `dataReduction` will be a bool variable)